### DISCUSSION TITLE
Cannot create thumbnails for large images

### DISCUSSION THREAD

**Discussion by damiano1976** [pimcore/pimcore#12419] [Q&A / Support] [answered]
Is there a way to increase the width/height limit for Imagemagick? If so, where would I find the settings to do so?

Pimcore Version: 10.0.7

Referred here by Bitnami Support: 
https://community.bitnami.com/t/running-c-script-on-server/106146

![Capture](https://user-images.githubusercontent.com/17390779/173592268-09d6881f-7fee-45f7-a55c-a601681355a5.PNG)

**Comment by kingjia90** [comment, marked as answer]
IINM Pimcore doesn't manage these settings, please have a try with this 
https://legacy.imagemagick.org/discourse-server/viewtopic.php?p=156413#p156413

**Reply by damiano1976** [reply]
Thank you @kingjia90 . That was exactly what I needed. Much appreciated!
